src/do_turn.cpp

```diff
@@ -663,6 +669,8 @@ bool do_turn()
     }
 
     // Process sound events into sound markers for display to the player.
+    music::deactivate_music_id( music::music_id::sound );
+
     sounds::process_sound_markers( &u );
```

Comment:
```suggestion
    music::deactivate_music_id( music::music_id::sound );

    // Process sound events into sound markers for display to the player.
    sounds::process_sound_markers( &u );
```

Reply:
Yeah, I think I placed that under the comment so as to accentuate the relationship between the two, but I guess it still does not fall under 'Process sound events'. Applied the change according to your suggestion.